src/main/java/org/fcrepo/spec/testsuite/crud/HttpPut.java

```diff
@@ -366,4 +367,71 @@ public void nonRDFSourcePutDigestResponseHeaderVerification(final String uri) {
                 .statusCode(400);
     }
 
+    /**
+     * 3.6.3-A
+     *
+     * @param uri The repository root URI
+     */
+    @Test(groups = {"MAY"})
+    @Parameters({"param1"})
+    public void putCreateRDFSource(final String uri) {
+        final TestInfo info = setupTest("3.6.3-A", "putCreateRDFSource",
+                "Implementations may accept HTTP PUT to create resources",
+                "https://fcrepo.github.io/fcrepo-specification/#http-put-create", ps);
+
+        final String location = joinLocation(uri, info.getId());
+
+        doPutUnverified(location)
+                .then()
+                .statusCode(201);
+    }
+
+    /**
+     * 3.6.3-B create non-rdf source
+     *
+     * @param uri The repository root URI
+     */
+    @Test(groups = {"MAY"})
+    @Parameters({"param1"})
+    public void putCreateNonRDFSource(final String uri) {
+        final TestInfo info = setupTest("3.6.3-B", "putCreateNonRDFSource",
+                "Implementations may accept HTTP PUT to create resources",
```

Comment:
Maybe indicate in the description that this tests creation of NonRDFSource by PUT.